console/account/how-to/use-the-search-bar.mdx

```diff
@@ -0,0 +1,37 @@
+---
+meta:
+  title: How to use the search bar
+  description: This page explains how to use the search bar
+content:
+  h1: How to use the search bar
+  paragraph: This page explains how to use the search bar
+tags: account search-bar resource
+dates:
+  validation: 2024-03-14
+  posted: 2024-03-14
+categories:
+  - account
+---
+
+The **search bar** functionality within the Scaleway console allows you to quickly find resources.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Log into the [Scaleway console](https://console.scaleway.com).
+2. Navigate to the search bar located next to the organization dashboard menu.
+    <Message type="tip">
+      You can also access the search bar by typing Cmd + k on macOS and Ctrl + k on Linux or Windows.
+    </Message>
+3. Click in the search bar and enter the keyword you are looking for. The keyword can be a Name or an IP of a resource.
```

Comment:
```suggestion
3. Click in the search bar and enter the keyword you are looking for. The keyword can be the name or the IP of a resource.
```

Reply:
Am I right in understanding that tags cannot be used in the search to find resources yet?

Reply:
> Am I right in understanding that tags cannot be used in the search to find resources yet?

yes :)